api/src/main/resources/leonardo.yaml

```diff
@@ -2133,7 +1624,9 @@ components:
           description: The user-supplied name for the runtime
         googleProject:
           type: string
-          description: The Google Project used to create the runtime
+          description: Deprecated. The Google Project used to create the runtime
+        cloudContext:
+          $ref: '#/components/schemas/CloudContext'
```

Comment:
Does this need a description?

Reply:
It is Leonardo's term.  We do have an entity description later in the Swagger (copied from Leo) "Cloud platform which holds context for a workspace" which is not super helpful